docs/UserGuide.md

```diff
@@ -417,18 +422,21 @@ not, `editCompany` has got you covered. Simply edit the parts of the company det
 
 **Function:** Edits the specified company within the displayed company list.
 
-**Format:** `editCompany COMPANY_INDEX [n/COMPANY_NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS]`
+**Format:** `editCompany COMPANY_INDEX n/COMPANY_NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS]`
 
 * Edits the company at the specified INDEX. The index refers to the index number shown in the displayed company list.
   The index must be a positive integer 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* The `PHONE_NUMBER`, `EMAIL` and `ADDRESS` here can be left blank, allowing you to remove previously added information which may be erroneous.
 * More restrictions for command parameters can be found [here](#c-company-restrictions)
 
 **Examples:**
 
 * `list` followed by `editCompany 1 p/91234567 e/johndoe@example.com`edits the 1<sup>st</sup> company in the displayed
   company list.
+* `list` followed by `editCompany 1 p/ e/`removes `PHONE_NUMBER` and `EMAIL` from the 1<sup>st</sup> company in the displayed
+  company list.
```

Comment:
Should this be part of the same line?

Reply:
what you mean? like `list` followed by `editCompany 1 p/91234567 e/johndoe@example.com`edits the 1<sup>st</sup> company in the displayed

Reply:
what you mean? like * `list` followed by `editCompany 1 p/91234567 e/`edits the 1<sup>st</sup>edits PHONE_NUMBER and removes EMAIL from the company in the displayed list?